frontend/src/app/utils/testResults.ts

```diff
@@ -56,6 +56,20 @@ export function hasCovidResults(results: MultiplexResults): boolean {
   );
 }
 
+export function hasDiseaseSpecificResults(
```

Comment:
⚠️ eventually we can refactor to use this instead of the covid specific version on `line 51`